_rules/important-letter-spacing-wide-enough-fsy8c7.md

```diff
@@ -0,0 +1,301 @@
+---
+id: fsy8c7
+name: Important letter spacing in style attributes is wide enough
+rule_type: atomic
+description: |
+  This rule checks that the `style` attribute is not used to prevent adjusting `letter-spacing` by using `!important`, except if it's at least 0.12 times the font size.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
```

Comment:
Regardless of if this is a new rule or if this changes the existing rule, I think Jey should be listed as a previous author.